server/routes/api.js

```diff
@@ -5,11 +5,54 @@ const router = require('express').Router(); // eslint-disable-line new-cap
 const debug = require('debug')('screens:api');
 const screens = require('../screens');
 const moment = require('moment');
-const fetch = require('node-fetch');
+const request = require('request');
 const transform = require('../urls');
 const transformedUrls = {};
 const log = require('../log');
 const pages = require('../pages');
+const curry = require('lodash').curry;
+
+// createJsonResponse :: Response-object -> Int -> Boolean -> Object -> IO
+function createJsonResponse (res, statusCode, success, data){
+  res.status(statusCode)
+  .json({
+    success,
+    data,
+    statusCode: code
+  });
+}
+
+const response = curry(createJsonResponse);
```

Comment:
It's begin used on line 31. Adding node-fetch back in too as it's used for the URL shortening code.